app/assets/stylesheets/layout/common/_header.scss

```diff
@@ -1,7 +1,7 @@
 .l-header {
-  background: $color-green-primary;
-  min-height: $baseline-unit*11;
-  position: relative;
+  width: 100%;
+  z-index: 200;
+  min-height: 55px;
```

Comment:
The actual height of the header is 30px + 1.5rem, i.e. is still a mix of pixels and rems so whether that is expressed in pixels or rems seems to me not to make much difference - still a bit imperfect either way.